### ISSUE TITLE
v2.4 strengthify missing in js files

### ISSUE THREAD

**Issue by mpselby** [CachetHQ/Cachet#3400]
Owing to a need to update PHP to 7.2 (previous going out of support) on CentOS 7 I have had to try the 2.4 branch after problems with laravel compatibility.  None of the javascripts sent to the browser define the $.strengthify() .

**Comment by CptTZ** [comment]
Same problem here, console shows:
```
Uncaught TypeError: $(...).strengthify is not a function
    at HTMLDocument.<anonymous> (all.js:27)
    at l (all.js:11)
    at Object.fireWith [as resolveWith] (all.js:11)
    at Function.ready (all.js:11)
    at HTMLDocument.a (all.js:10)
```
Which leads to incorrect renderings for some components on the status page.

**Comment by jbrooksuk** [comment]
Thanks for reporting! I'll take a look now.

**Comment by jbrooksuk** [comment]
Ah, we removed Strengthify in https://github.com/CachetHQ/Cachet/commit/9de2ea22e3b25ab402317c83dd605d6c8a83f548 due to it being third-party hosted, but it looks like I forgot to remove the actual integration. I'll do that now.

**Comment by jbrooksuk** [comment]
Context is at https://github.com/CachetHQ/Cachet/pull/3315

**Comment by jbrooksuk** [comment]
Fixed in c0d9d9ceb80b43c6e566666ad74173b0a42725f7
